--- dolby_vision-3.3.0/Cargo.toml	1970-01-01T00:00:01+00:00
+++ dolby_vision-3.3.0/Cargo.toml	2024-04-19T13:18:11.600532+00:00
@@ -39,10 +39,7 @@
 
 [lib]
 doctest = false
-
-[[bench]]
-name = "bench_main"
-harness = false
+crate-type = ["lib", "cdylib"]
 
 [dependencies.anyhow]
 version = "1.0.81"
@@ -76,9 +73,6 @@
 features = ["preserve_order"]
 optional = true
 
-[dev-dependencies.criterion]
-version = "0.5.1"
-
 [features]
 capi = ["libc"]
 serde = [
